src/services/Store/store/asset.slice.test.ts

```diff
@@ -67,3 +73,16 @@ describe('AccountSlice', () => {
     expect(actual).toEqual(initialState);
   });
 });
+
+describe('fetchAssetsWorker()', () => {
+  expectSaga.DEFAULT_TIMEOUT = 100;
```

Comment:
I guess it is useful for all our saga tests.
In this case can we set the default in `test-utils` and import from there ?